src/components/PeopleList.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useEffect, useState } from 'react';
+import { Person } from '../types';
+import { PeopleItem } from './PeopleItem';
+
+type Props = {
+  people: Person[],
+};
+
+export const PeopleList: React.FC<Props> = ({
+  people,
+}) => {
+  const [isNoPeople, setIsNoPeople] = useState(false);
+
+  useEffect(() => {
+    if (!people.length) {
+      setIsNoPeople(true);
+    }
+  }, [people]);
```

Comment:
we don't need state for this logic

let's just check the people length in some condition